build.gradle

```diff
@@ -25,20 +24,6 @@ allprojects {
     }
 }
 
-// TODO Remove once either Facebook or Robolectric fix the crash: https://github.com/robolectric/robolectric/issues/2579#issuecomment-276526911
-gradle.taskGraph.whenReady { graph ->
```

Comment:
@samtstern It looks like the commit that fixed this didn't end up in the new Facebook release anyway unfortunately. As for the other issue, I believe that was unrelated though I'm not sure...